### ISSUE TITLE
NPE for gisgraphy provider

### ISSUE THREAD

**Issue by karussell** [graphhopper/geocoder-converter#45]
@gisgraphy I'm getting for a simple "q=dresden" the following:
```
! java.lang.NullPointerException: null
! at com.graphhopper.converter.resources.ConverterResourceGisgraphy.buildBaseTarget(ConverterResourceGisgraphy.java:154)
! at com.graphhopper.converter.resources.ConverterResourceGisgraphy.buildGeocodingTarget(ConverterResourceGisgraphy.java:124)
! at com.graphhopper.converter.resources.ConverterResourceGisgraphy.handle(ConverterResourceGisgraphy.java:85)
```

**Comment by karussell** [comment]
Ah, that's the result if the API key is empty ...

**Comment by gisgraphy** [comment]
The apikey it should not be required, I have fixed it 537b8109ac6bf580d50035ebead199d955cc9b4c (included in PR)

**Comment by karussell** [comment]
Thanks!

**Comment by boldtrn** [comment]
Is this issue fixed now and we can close it or does it still exist?

**Comment by karussell** [comment]
Fixed with #44
